webpack.config.js

```diff
@@ -1,67 +1,70 @@
-const StyleLintPlugin = require('stylelint-webpack-plugin');
-const packageJSON = require('./package.json');
-const path = require('path');
+const StyleLintPlugin = require("stylelint-webpack-plugin");
+const packageJSON = require("./package.json");
+const path = require("path");
+const ESLintPlugin = require("eslint-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
 
 const DIST = path.resolve("./dist");
-const MODE = process.env.NODE_ENV || 'development';
-const SOURCE_MAP = 'sourec-map';
-const DEVTOOL = (process.env.NODE_ENV === 'development') ? SOURCE_MAP : false;
+const MODE = process.env.NODE_ENV || "development";
+const SOURCE_MAP = "source-map";
+const DEVTOOL = process.env.NODE_ENV === "development" ? SOURCE_MAP : false;
 
-console.log('Webpack mode:', MODE); // eslint-disable-line no-console
+console.log("Webpack mode:", MODE); // eslint-disable-line no-console
+
+const eslintPluginOptions = {
+  // enforce: "pre",
+  // test: /\.js?$/,
+  extensions: ["js", "jsx", "ts"],
+  exclude: ["/(node_modules)/", "/Library/"],
+  failOnError: true,
+};
 
 const config = {
-  devtool: DEVTOOL,
-  entry: [
-    './src/index.js'
-  ],
+  // Hard coded to be able to get the source map file
+  // devtool: DEVTOOL,
+  devtool: SOURCE_MAP,
```

Comment:
Apparently `process.env.NODE_ENV` is always undefined and I get devtool false here so I just hard coded the source map to be sure it creates the .map file.

Reply:
We don't want the source maps in production. So this shouldn't be merged. We would need a different way of adding the source maps, possibly through a `build:dev` command

Reply:
Ok, got it! will fix it!